src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/helper/builders/IncentiveRepository.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.helper.builders
+
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Repository
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.jpa.repository.IncentiveMappingRepository
+
+@Repository
+@Transactional
+class IncentiveRepository(@Qualifier("incentiveMappingRepository") private val incentiveRepository: IncentiveMappingRepository) : IncentiveMappingRepository by incentiveRepository
```

Comment:
Not sure what this does .. is it referenced anywhere?

Reply:
There are two IncentiveMappingRepository beans and unless I put a qualifier in - I got a compilation error. I'm sure there must be a way around it and will have another look